.github/workflows/tox-pytest.yml

```diff
@@ -5,45 +5,58 @@ on: [push, pull_request]
 jobs:
   ci-test:
     runs-on: ubuntu-latest
-    permissions:
-      contents: read
-      id-token: write
     strategy:
       fail-fast: false
+    permissions:
+      contents: "read"
+      id-token: "write"
+    defaults:
+      run:
+        shell: bash -l {0}
 
     steps:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 2
 
       - name: Set up conda environment for testing
-        uses: conda-incubator/setup-miniconda@v3.0.4
+        uses: mamba-org/setup-micromamba@v1
         with:
-          miniforge-variant: Mambaforge
-          miniforge-version: latest
-          use-mamba: true
-          mamba-version: "*"
-          channels: conda-forge,defaults
-          channel-priority: true
-          python-version: "3.10"
-          activate-environment: pudl-usage-metrics
           environment-file: environment.yml
-      - shell: bash -l {0}
+          cache-environment: true
+          cache-environment-key: environment-${{ hashFiles('pyproject.toml') }}
+          condarc: |
+            channels:
+            - conda-forge
+            - defaults
+            channel_priority: strict
+
+      - name: Log conda environnment information
         run: |
-          mamba info
-          mamba list
+          conda info
+          conda list
```

Comment:
Why did you switch from mamba to conda?

Reply:
I was mirroring what we do in PUDL and the pudl archiver repos.